doc/index.md

```diff
@@ -1,100 +1,178 @@
 # Introduction
 
-lua-http is an HTTP library for Lua, it supports: both client and server operations, both HTTP 1 and HTTP 2.
+Lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. Lua-http can be run in either a Server (listen) or Client (connect) mode and includes the following features: 
+- HTTP 1 and HTTP 2 as specified by RFCs [1945](https://tools.ietf.org/html/rfc1945) and [7540 ](https://tools.ietf.org/html/rfc7540)
+- WebSockets as specified by RFCs [6455](https://tools.ietf.org/html/rfc6455) and [7936](https://tools.ietf.org/html/rfc7936) including ping/pong, binary data transfer and TLS encyrption
+- Fully event driven model with low resource overhead
+- Transport Layer Security (TLS)- Lua-http supports HTTPS and WSS via openssl.  
+- Easy integration into other event driven application models
 
-HTTP over TLS (i.e. HTTPS) is fully supported.
+Lua-http was written to fill a gap in the Lua ecosystem by providing event driven HTTP and WebSocket capabilities to Lua applications without forcing the developer to follow a specific pattern, or even know anything about the transportation of data (other than the connection addresses). The library is simple and un-obtrusive and is designed to accommodate tens of thousands of connections on commodity hardware. This allows other developers to concentrate on line-of-business features when building Internet enabled applications, instead of worrying about the details of connection management. 
 
 
-## Conventions
+### Common use cases
```

Comment:
Should never go from top level `#` header to level 3 `###` header. I'm not sure why this isn't level 2?